### ISSUE TITLE
gstreamer 1.12.5 needed for 4.0.0

### ISSUE THREAD

**Issue by prometheanfire** [wwmm/pulseeffects#224]
This wasn't called out in the changelog, but is there a reason for this?  we (gentoo) don't have this packaged yet.

**Comment by wwmm** [comment]
Unfortunately there is a reason https://bugzilla.gnome.org/show_bug.cgi?id=791330. Previous GStreamer versions have a bug where lv2 plugins like the ones from Calf Studio have their boolean variables values inverted. In PE 3.x series I had applied  workarounds for that bug in many places but while I was rewriting PE in C++ it became very inconvenient to keep them there. They were not a good fit for the code design I had in mind. So I dropped them.

PE will probably compile with older GStreamer versions but all plugins that are accessed through LV2 are not going to work (Calf and Linux Studio Plugins). The equalizer is a native plugin and is not affected by this bug.
